hls4ml/converters/keras/convolution.py

```diff
@@ -84,5 +84,8 @@ def parse_conv2d_layer(keras_layer, input_names, input_shapes, data_reader, conf
         output_shape = [input_shapes[0][0], layer['n_filt'], layer['out_height'], layer['out_width']]
     else:
         output_shape = [input_shapes[0][0], layer['out_height'], layer['out_width'], layer['n_filt']]
-
+    if layer['class_name'] == 'QConv2DBatchnorm':
+        intermediate_shape = list()
+        intermediate_shape.append(output_shape)
+        output_shape = intermediate_shape
```

Comment:
Why is this needed?

Reply:
Because for `QConv2DBatchnorm` `parse_conv2d_layer()` and `parse_batchnorm_layer()` are called one after the other. Without embedding the shape in a 2D-vector here, it will result in an error when `parse_batchnorm_layer()` executes, see line 111 of keras/core.py ( `for dim in input_shapes[0][1:]:` ). Otherwise I can add a control inside the `parse_batchnorm_layer()` method that checks if the considered layer is `QConv2DBatchnorm` and modify the way it access `input_shapes` field.